requirements.in

```diff
@@ -0,0 +1,93 @@
+aboutcode-toolkit==4.0.1
```

Comment:
If you generated this with pip-tools it would be nice to have the annotate options that tells who is a dependency introduced by whom

Reply:
No, i generated using pip freeze that you told me during session.

Reply:
no , its pip freeze>requirements.in